tests/state/_common/actions/setActiveKey-test.js

```diff
@@ -0,0 +1,40 @@
+import commonActions from '../../../../src/state/_common/actions';
+import testBatchRunner from '../../helpers';
+import {commonActionTypesObj as actionTypes} from '../../../constants';
+
+const tests = [
+	{
+		name: 'It dispatch setActiveKey.',
+		action: (actions, actionTypes, options) => {
+			return (dispatch, getState) => {
+				let action;
+				// for common testing
+				if (actionTypes && options) {
```

Comment:
Is options check necessary?

Reply:
Yes, existence of actionTypes and options means that tests is for common action not implemented in some store.